tests/kohana/RequestTest.php

```diff
@@ -736,10 +736,60 @@ public function test_passes_client_params()
 		$this->assertEquals($client->strict_redirect(), FALSE);
 	}
 
+	/**
+	 * Tests correctness request content-length header after calling render
+	 */
+	public function test_content_length_after_render()
+	{
+		$request = Request::factory('https://example.org/post')
+			->client(new Request_Client_Content_Length_Kohana_RequestTest)
+			->method(Request::POST)
+			->post(array('aaa' => 'bbb'));
+
+		$request->render();
+
+		$response = $request->execute();
+
+		$this->assertEquals(strlen($request->body()), $response->body());
```

Comment:
@acoulton I was reviewing this PR and I just found your note here! It had slipped away from my attention all this time, I am sorry.